seed/static/seed/js/services/dataset_service.js

```diff
@@ -43,12 +43,11 @@ angular.module('BE.seed.service.dataset', []).factory('dataset_service', [
     };
 
     dataset_service.delete_file = function (file_id) {
-      return $http.delete(window.BE.urls.delete_file, {
+      return $http.delete('/api/v2/import_files/' + file_id + '/', {
```

Comment:
I know there were really good intentions behind the window.BE.urls shortcuts, but I actually like the explicit nature of this better.  Not sure why.